rate-providers/sweepRateProvider.md

```diff
@@ -4,10 +4,10 @@
 - Reviewed by: @mkflow27
 - Checked by: @danielmkm
 - Deployed at:
-    - [ethereum:0x9cE2837d6d84bb521A5a3002a2132B9E9E9cc4C8](https://etherscan.io/address/0x9cE2837d6d84bb521A5a3002a2132B9E9E9cc4C8#code)
+    - [ethereum:0xe3E123ED9fec48a6f40A8aC7bE9afEDDAD80F146](https://etherscan.io/address/0xe3E123ED9fec48a6f40A8aC7bE9afEDDAD80F146#code)
     - [arbitrum:0x3bB6861c0Be6673809D55b9D346b6774B634a9D7](https://arbiscan.io/address/0x3bB6861c0Be6673809D55b9D346b6774B634a9D7)
-    - [optimism:0x33A48e4aA79A66fc4f7061f5D9E274528C213029](https://optimistic.etherscan.io/address/0x33A48e4aA79A66fc4f7061f5D9E274528C213029)
-    - [avalanche:0x709d075147a10495e5c3bBF3dfc0c138F34C6E72](https://snowtrace.dev/address/0x709d075147a10495e5c3bBF3dfc0c138F34C6E72/contract/43114/code)
+    - [optimism:0xff368E106EA8782FaB6B2D4AD69739a60C66400E](https://optimistic.etherscan.io/address/0x33A48e4aA79A66fc4f7061f5D9E274528C213029)
```

Comment:
URL is not updated.

Reply:
@wei3erHase thanks for spotting. I have corrected it.